tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -285,7 +285,7 @@ public function test_disputes_notification_badge_display() {
 		$this->assertArrayHasKey( $dispute_url, $item_names_by_urls );
 
 		// The expected badge content should include 4 disputes needing a response.
-		$expected_badge = sprintf( WC_Payments_Admin::DISPUTE_NOTIFICATION_BADGE_FORMAT, 4 );
+		$expected_badge = sprintf( WC_Payments_Admin::UNRESOLVED_NOTIFICATION_BADGE_FORMAT, 4 );
 
 		$this->assertEquals( 'Disputes' . $expected_badge, $item_names_by_urls[ $dispute_url ] );
```

Comment:
Since we are touching this, can we use strict assert here?

Reply:
Fixed in 2090c06.